lib/ts/recipe/emailpassword/utils.ts

```diff
@@ -81,6 +108,14 @@ export function validateAndNormaliseEmailVerificationConfig(
     config?: TypeInput
 ): TypeNormalisedInputEmailVerification {
     return {
+        emailDelivery:
+            config?.emailDelivery === undefined
+                ? undefined
+                : {
+                      service: config.emailDelivery.service,
+                      // override is undefined here intentionally because the user's override
+                      // for email verification will override the default implementation for emailpassword recipe. So this is not needed.
+                  },
```

Comment:
This can just not be specified at all.

Reply:
We need it cause if we don't pass it, then the emailveirfication API will call it's default implementation which will call our API instead of using the service passed by the user.